carbon/components/PageHeader/PageHeader.tsx

```diff
@@ -39,7 +40,11 @@ export const PageHeader: FC<{
       <div className={layout.desktopOnly}>
         <div className={styles.buttons}>
           <ChangeLanguageButton className={styles.changeLanguageButton} />
-          <Button className={styles.exploreButton}>Explore Marketplace</Button>
+          <Link href="https://www.carbonmark.com/">
```

Comment:
let's use the link from the constants here

Reply:
done in 82bbaa89